Sources/swift-format/Run.swift

```diff
@@ -76,7 +75,7 @@ func lintMain(
 func formatMain(
   configuration: Configuration, sourceFile: FileHandle, assumingFilename: String?, inPlace: Bool,
   debugOptions: DebugOptions, diagnosticEngine: DiagnosticEngine
-) -> Int {
+) {
   let formatter = SwiftFormatter(configuration: configuration, diagnosticEngine: diagnosticEngine)
```

Comment:
I recently changed this to pass nil, so that `DiagnosticEngine` is only used for fatal messages when formatting, in #155. It looks like this is causing a merge conflict - can you update and resolve the conflict?